app/assets/stylesheets/layout_components/reveal-modal.scss

```diff
@@ -20,7 +20,17 @@ dialog .close-reveal-modal{
     font-weight: normal;
   }
 
+  @include breakpoint(medium-up){
```

Comment:
I think (medium-up) is old syntax from Foundation 5. Foundation 6 should (medium up) or just (medium) for screen sizes medium and larger. Once we upgrade to 6.3 this will break.